source/brailleInput.py

```diff
@@ -378,39 +378,76 @@ class BrailleInputGesture(inputCore.InputGesture):
 	space = False
 
 	def _makeDotsId(self):
-		# Items separated by a plus sign need to be in Python set order.
-		items = {"dot%d" % (i+1) for i in xrange(8) if self.dots & (1 << i)}
+		items = ["dot%d" % (i+1) for i in xrange(8) if self.dots & (1 << i)]
 		if self.space:
-			items.add("space")
+			items.append("space")
 		return "bk:" + "+".join(items)
 
+	#: The generic gesture identifier for space plus any dots.
+	#: This could be used to bind many braille commands to a single script.
+	GENERIC_ID_SPACE_DOTS = inputCore.normalizeGestureIdentifier("bk:space+dots")
+	#: The generic gesture identifier for any dots.
+	#: This is used to bind entry of braille text to a single script.
+	GENERIC_ID_DOTS = inputCore.normalizeGestureIdentifier("bk:dots")
+
 	def _get_identifiers(self):
 		if self.space and self.dots:
-			# Items separated by a plus sign need to be in Python set order.
-			generic = "bk:" + "+".join({"space", "dots"})
-			return (self._makeDotsId(), generic)
+			return (self._makeDotsId(), self.GENERIC_ID_SPACE_DOTS)
 		elif self.dots in (DOT7, DOT8, DOT7 | DOT8):
 			# Allow bindings to dots 7 and/or 8 by themselves.
-			return (self._makeDotsId(), "bk:dots")
+			return (self._makeDotsId(), self.GENERIC_ID_DOTS)
 		elif self.dots or self.space:
-			return ("bk:dots",)
+			return (self.GENERIC_ID_DOTS,)
 		else:
 			return ()
 
-	def _get_displayName(self):
-		if not self.dots and not self.space:
-			return None
-		# Translators: Reported before braille input in input help mode.
-		out = _("braille") + " "
-		if self.space and self.dots:
+	@classmethod
+	def _makeDisplayText(cls, dots, space):
+		if space and dots:
 			# Translators: Reported when braille space is pressed with dots in input help mode.
-			out += _("space with dot")
-		elif self.dots:
+			out = _("space with dot")
+		elif dots:
 			# Translators: Reported when braille dots are pressed in input help mode.
-			out += _("dot")
-		elif self.space:
+			out = _("dot")
+		elif space:
 			# Translators: Reported when braille space is pressed in input help mode.
-			out += _("space")
-		if self.dots:
-			out += " " + formatDotNumbers(self.dots)
+			out = _("space")
+		if dots:
+			out += " " + formatDotNumbers(dots)
 		return out
+
+	def _get_displayName(self):
+		if not self.dots and not self.space:
+			return None
+		return self._makeDisplayText(self.dots, self.space)
+
+	@classmethod
+	def getDisplayTextForIdentifier(cls, identifier):
```

Comment:
Perhaps a docstring on this to explain what its for? Also, it would be nice to see a couple of examples of what `identifier` might be.